example_configs/one_of_every_device_offline.yaml

```diff
@@ -53,10 +53,60 @@ buses:
       high_pos_cmd_limit_eu:            3.14159
       high_pos_cal_limit_eu:            3.2 
       holding_duration_sec:            3.0 # chosen arbitrarily
-      egd_brake_engage_msec:           500 
-      egd_brake_disengage_msec:        1000
-      egd_crc:                         12345
-      egd_drive_max_current_limit:     10 # amps
```

Comment:
The bus topologies will be broken anyway because of the renaming these parameters, why not using the proper name in `device_class` then? Better to fix the bus topology in a single pass than in phases, in my opinion.